src/VisualStudio/TestUtilities/InProcess/InteractiveWindow_InProc.cs

```diff
@@ -0,0 +1,173 @@
+
+            // find last prompt and remove
+            int lastPromptIndex = replText.LastIndexOf(ReplPromptText);
+
+            if (lastPromptIndex > 0)
+            {
+                replText = replText.Substring(0, lastPromptIndex);
+            }
+
+            // it's possible for the editor text to contain a trailing newline, remove it
+            return replText.EndsWith(Environment.NewLine)
+                ? replText.Substring(0, replText.Length - Environment.NewLine.Length)
+                : replText;
+        }
+
+        /// <summary>
+        /// Gets the last output from the REPL.
+        /// </summary>
+        public string GetLastReplOutput()
+        {
+            // TODO: This may be flaky if the last submission contains ReplPromptText
+
+            var replText = GetReplTextWithoutPrompt();
+            int lastPromptIndex = replText.LastIndexOf(ReplPromptText);
+
+            replText = replText.Substring(lastPromptIndex, replText.Length - lastPromptIndex);
+            int lastSubmissionIndex = replText.LastIndexOf(ReplSubmissionText);
+
+            if (lastSubmissionIndex > 0)
+            {
+                replText = replText.Substring(lastSubmissionIndex, replText.Length - lastSubmissionIndex);
+            }
+            else if (!replText.StartsWith(ReplPromptText))
+            {
+                return replText;
+            }
+
+            int firstNewLineIndex = replText.IndexOf(Environment.NewLine);
+
+            if (firstNewLineIndex <= 0)
+            {
+                return replText;
+            }
+
+            firstNewLineIndex += Environment.NewLine.Length;
+            return replText.Substring(firstNewLineIndex, replText.Length - firstNewLineIndex);
+        }
+
+        public void Reset(bool waitForPrompt = true)
+        {
+            ExecuteCommand(ResetCommand);
+
+            if (waitForPrompt)
+            {
+                WaitForReplPrompt();
+            }
+        }
+
+        public void SubmitText(string text, bool waitForPrompt = true)
+        {
+            _interactiveWindow.SubmitAsync(new[] { text }).Wait();
+
+            if (waitForPrompt)
+            {
+                WaitForReplPrompt();
+            }
+        }
+
+        public void CloseWindow()
+        {
+            var dte = GetDTE();
+
+            foreach (EnvDTE.Window window in dte.Windows)
+            {
+                if (window.Caption == _windowTitle)
+                {
+                    window?.Close();
+                    break;
+                }
+            }
+        }
+
+        public void ShowWindow(bool waitForPrompt = true)
+        {
+            ExecuteCommand(_viewCommand);
+
+            if (waitForPrompt)
+            {
+                WaitForReplPrompt();
+            }
+        }
+
+        public void WaitForReplPrompt()
+        {
+            WaitForReplPromptAsync().ConfigureAwait(continueOnCapturedContext: false).GetAwaiter().GetResult();
```

Comment:
The .ConfigureAwait() is unnecessary here. I suspect you're moving code around, so apologies for the nitpick if that's the case.